src/fhirConfig.ts

```diff
@@ -23,8 +23,21 @@ export interface OAuthStrategy {
     revokeUrl?: string;
 }
 
+/**
+ * http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#using-well-known
+ */
+export interface SmartStrategy extends OAuthStrategy {
+    capabilities: string[];
```

Comment:
Can we make this a type? http://www.hl7.org/fhir/smart-app-launch/conformance/index.html#core-capabilities or is this too rigid / incomplete url?

Reply:
In terms of one-way-door vs two-way-door decisions, I'm thinking keeping it a `string[]` is a two way door decision while making it into a specific type is a one way door decision. 

Leaving it as a `string[]` now will still allow us to narrow the scope in the future if we get customer feedback for that in the future. That will only be a minor version bump. However, scoping it down now will require a major version bump in the future as that is breaking change.

Reply:
+1 good thought

Reply:
@rsmayda , @nguyen102 you can do a combination of both like:

```ts
export type capability = "launch-ehr"|"launch-standalone"|"etc..."|string;
```
functionally that type is equivalent to `string` since any string literal is contained in `string`, but expressing it this way helps document the intended usage and allows Typescript to do some autocompletion on your IDE.

I've seen that pattern often used on the aws-sdk. i.e. https://github.com/aws/aws-sdk-js/blob/master/clients/glue.d.ts#L4826